generator-api/src/main/scala/com/lightbend/lagom/spec/ResourceUtils.scala

```diff
@@ -42,4 +43,25 @@ object ResourceUtils {
     }
   }
 
+  /**
+   * @param folder an absolute folder
+   * @param filename a relative path to a file (may contain File.separators)
+   * @param fileContents
+   * @return
+   */
+  def writeFile(folder: File, filename: String, fileContents: String): File = {
+    val path = Paths.get(folder.getAbsolutePath, filename)
+    val theFolder = path.toFile.getParentFile
+    while (!theFolder.exists()) {
```

Comment:
The loop should not have been committed, it was an attempt to debug indirectly.
When running this code from a `scripted` test I couldn't find a way to debug and for some reason the directories were not built. Causing the infinite loop was a way to detect if something fishy was going on there.

Thanks for warning!